lms/envs/common.py

```diff
@@ -2023,6 +2023,8 @@
     'WHAT_IS_VERIFIED_CERT': 'verified-certificate',
 }
 
+SUPPORT_SITE_LINK = ''
```

Comment:
Yes -- the logic for showing the link only cares if `bool(SUPPORT_SITE_LINK) == False`, and I see no reason to set a default of a type that the code doesn't expect.